src/quo2/screens/main.cljs

```diff
@@ -82,16 +82,7 @@
            :component info-message/preview-info-message}
           {:name      :information-box
            :insets    {:top false}
-           :component information-box/preview-information-box}
-          {:name      :lowest-price
-           :insets    {:top false}
-           :component lowest-price/preview-lowest-price}
-          {:name      :network-breakdown
-           :insets    {:top false}
-           :component network-breakdown/preview-network-breakdown}
-          {:name      :network-amount
-           :insets    {:top false}
-           :component network-amount/preview}]
+           :component information-box/preview-information-box}]
```

Comment:
@ibrkhalil Please don't put components in random directories. We use Figma like directory structure so that components are easy to find. Thank you

Reply:
Cool Parvesh, Thanks
I'll use Figma structure to put them in their correct directories